apps/condo/domains/settings/access/MobileFeatureConfig.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyGreaterThanPreviousMeterReadingIsEnabled:Checkbox; meta:Json; ticketSubmittingIsEnabled:Checkbox'`
+ */
+
+const { uniq, map, get } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find, getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadMobileFeatureConfigs ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === RESIDENT) {
+        const residents = await find('Resident', { user: { id: user.id } })
+        const organizations = uniq(map(residents, 'organization'))
+
+        if (residents.length > 0) {
+            return {
+                organization: { id_in: organizations },
+                deletedAt: null,
+            }
+        }
+        return false
+    }
+
+    return {
+        organization: {
+            OR: [
+                queryOrganizationEmployeeFor(user.id),
+                queryOrganizationEmployeeFromRelatedOrganizationFor(user.id),
+            ],
+        },
+    }
```

Comment:
Yes, and I think that's right. The `organization` settings page is available to everyone at the moment. Special rights are only used to check editing rights. I think it will be correct to repeat the behavior of the other sections of the `organization` settings